pkgs/development/compilers/go-jsonnet/default.nix

```
@@ -13,12 +13,12 @@ buildGoModule rec {
 
   vendorSha256 = "0nsm4gsbbn8myz4yfi6m7qc3iizhdambsr18iks0clkdn3mi2jn1";
 
-  subPackages = [ "cmd/jsonnet" ];
+  subPackages = [ "cmd/jsonnet", "cmd/jsonnetfmt" ];
```

list items are separated by space, not comma https://nixos.org/nix/manual/#idm140737322008768
```suggestion
  subPackages = [ "cmd/jsonnet" "cmd/jsonnetfmt" ];
```
besides that, LGTM